### PR TITLE
Filter out leaderboard pages without donations

### PR DESCRIPTION
**Pull request by leonfu103**

### CODE REVIEW

File: src/components/mixins/Leaderboard.js [hunk 1]
```diff
@@ -56,17 +52,30 @@ module.exports = {
...
-      id: page.id,
-      name: page.name,
-      url: page.url,
-      isoCode: page.amount.currency.iso_code,
-      amount:  page.amount.cents,
-      totalMembers: page.team_member_uids.length,
-      imgSrc: page.image.large_image_url,
-      medImgSrc: page.image.medium_image_url
-    };
+  CheckHasContentCallback: function(leaderboard) {
```
Comment (davidherse) [inline src/components/mixins/Leaderboard.js]: Lowercase C
Also, I'm just wondering why you broke this out into a separate method.

Comment (leonfu103) [inline src/components/mixins/Leaderboard.js]: @davidherse It is easier to test the `onHasContent` callback. Before I had to compose a leaderboard object to pass in to the processLeaderboard method. Now I only need an array `[1]`.

File: src/components/mixins/Leaderboard.js [hunk 2]
```diff
@@ -56,17 +52,30 @@ module.exports = {
...
-      id: page.id,
-      name: page.name,
-      url: page.url,
-      isoCode: page.amount.currency.iso_code,
-      amount:  page.amount.cents,
-      totalMembers: page.team_member_uids.length,
-      imgSrc: page.image.large_image_url,
-      medImgSrc: page.image.medium_image_url
-    };
+  checkHasContentCallback: function(leaderboard) {
```
Comment (davidherse) [inline src/components/mixins/Leaderboard.js]: I'm just wondering why this is broken out into a separate method. Is it for tesitng?

Comment (leonfu103) [inline src/components/mixins/Leaderboard.js]: @davidherse Yes, it's only for testing.

**Pull request merged**